src/test/groovy/TestIdeaExtPlugin.groovy

```diff
@@ -235,7 +237,7 @@ task printSettings {
                                 {
                                     "type": "FILE",
                                     "sourceFiles": [
-                                        "${buildFile.canonicalPath.replace('\\\\' as char, '/' as char)}"
+                                        "${escapeJava(buildFile.canonicalPath.replace('\\\\' as char, '/' as char))}"
```

Comment:
I'd expect escapeJavaScript.

But in any case I have to wonder: Is there anything escapeJava can output which is not valid JSON?

For example, `\177` is a valid Java escape but not a valid JSON escape... but it probably doesn't matter because `escapeJava` _probably_ doesn't output it.